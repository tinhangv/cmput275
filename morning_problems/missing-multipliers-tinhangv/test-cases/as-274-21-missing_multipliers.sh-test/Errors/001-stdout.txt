/home/victor/Documents/c275/morning_problems/missing-multipliers-tinhangv/soln/.build/build.sh: line 8: 113378 Floating point exception(core dumped) ./$NAME

Expected file "test-cases/as-274-21-missing_multipliers.sh-test/Expected/001-stdout.txt" has no output file match
