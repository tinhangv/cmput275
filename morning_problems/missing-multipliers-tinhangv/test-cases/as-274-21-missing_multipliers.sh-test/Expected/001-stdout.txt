0 4 0 3 -6
